src/upload-tests.proj

```diff
@@ -7,20 +7,27 @@
          - Add support for storing test-project-specific timeouts, use this instead of default if available
   -->
 
-  <UsingTask TaskName="ZipFileCreateFromDirectory"       AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <UsingTask TaskName="SimulateHelixExecution"     AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.CloudTestTasks.dll"/>
+  <UsingTask TaskName="ZipFileCreateFromDirectory" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
 
   <PropertyGroup>
     <!-- Workaround for dealing with building on Linux/OSX.  
          It doesn't seem possible to pass a connection string on the command line, as it contains ';' characters. 
          Checking for the existence of a connection string will happen inside CloudTest.Helix.targets -->
     <CloudDropConnectionString    Condition="'$(CloudDropConnectionString)'==''"   >DefaultEndpointsProtocol=https;AccountName=$(CloudDropAccountName);AccountKey=$(CloudDropAccessToken);EndpointSuffix=core.windows.net</CloudDropConnectionString>
     <CloudResultsConnectionString Condition="'$(CloudResultsConnectionString)'==''">DefaultEndpointsProtocol=https;AccountName=$(CloudResultsAccountName);AccountKey=$(CloudResultsAccessToken);EndpointSuffix=core.windows.net</CloudResultsConnectionString>
-    
+
+    <!-- Set this to true to simulate Helix execution.  Will fail if Python 2.7.X is unavailable 
+         We'll also opportunistically set SkipSendToHelix true here, since if the user is doing 
+         local execution they likely don't want to run in the cloud. -->
+    <LocalHelixExecution>false</LocalHelixExecution>
```

Comment:
I thought about that but decided that a condition really only makes sense here in the case where we're calling this project from another project.  If provided on the command line, like /p:LocalHelixExecution=true, it will supersede this value.  If not provided, I want to make it as hard as possible for this to happen as it can be very slow (say, for an ILC run).

I'm happy to put in a check if you like, just didn't seem to matter for how this project is used.